src/components/LocationSelect.js

```diff
@@ -92,10 +92,6 @@ class LocationSelect extends React.Component {
               States
```

Comment:
Let's remove this option as well. That way the default selected option isn't "States" which is sort of confusing since there really aren't any alternatives.

![screen shot 2017-11-27 at 3 00 13 pm](https://user-images.githubusercontent.com/780941/33286558-b8ee79f8-d383-11e7-9eaf-9b327c484500.png)